### PR TITLE
Fixes #5468 - SSH second-hop for 7.1

### LINKED ISSUE #5468: Need to document new second hop behavior for interactive remote sessions
# Documentation Issue
This is related to PR: https://github.com/PowerShell/PowerShell/pull/11920

Interactive remoting second hop has been restricted to work only form within a WinRM session, but now works for any interactive remoting session.

## Context of the issue

- URL to the article (Docs or GitHub URL):
- Issue affects the following content (check all that apply):

  **Cmdlet reference & about_ topics**
  - [x] Version 7.x preview content
  - [ ] Version 7.0 content
  - [ ] Version 6 content
  - [ ] Version 5.1 content

  **Conceptual articles**
  - [x] Fundamental conceptual articles
  - [ ] Script sample articles
  - [ ] DSC articles
  - [ ] Gallery articles
  - [ ] JEA articles
  - [ ] WMF articles
  - [ ] SDK articles
- Is the issue specific to a platform (Y/N - Win/macOS/Linux):

### PR DESCRIPTION
**Pull request by sdwheeler**
# PR Summary

Fixes [AB#1752251](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1752251) - Fixes #5468 - SSH second-hop for 7.1

## PR Context

Select the area of the Table of Contents containing the documents being changed.

**Conceptual content**
- [ ] Overview and Install
- [ ] Learning PowerShell
  - [ ] PowerShell 101
  - [ ] Deep dives
  - [x] Remoting
- [ ] Release notes (What's New)
- [ ] Windows PowerShell
  - WMF, ISE, release notes, etc.
- [ ] DSC articles
- [ ] Community resources
- [ ] Sample scripts
- [ ] Gallery articles
- [ ] Scripting and development
  - [ ] Legacy SDK

**Cmdlet reference & about_ topics**
- [x] Version 7.1 preview content
- [ ] Version 7.0 content
- [ ] Version 6 content
- [ ] Version 5.1 content

## PR Checklist

- [x] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords][key].
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[WIP]` to the beginning of the
    title and remove the prefix when the PR is ready.

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

File: reference/7.1/Microsoft.PowerShell.Core/Enter-PSSession.md [hunk 1]
```diff
@@ -868,33 +868,42 @@ The cmdlet does not return any output.
...
+`Enter-PSSession` requires the `Get-Command`, `Out-Default`, and `Exit-PSSession` cmdlets. If these
+cmdlets are not included in the session configuration on the remote computer, the `Enter-PSSession`
+commands fails.
+
+Unlike `Invoke-Command`, which parses and interprets the commands before it sends them to the remote
+computer, `Enter-PSSession` sends the commands directly to the remote computer without
+interpretation.
+
+If the session you want to enter is busy processing a command, there might be a delay before
+PowerShell responds to **the Enter-PSSession** command. You will be connected as soon as the session
```
Comment (chasewilson) [inline reference/7.1/Microsoft.PowerShell.Core/Enter-PSSession.md:891]: Wast "the Enter-PSSession" meant to be bolded this way or should it be code fenced like the line below?

File: reference/7.1/Microsoft.PowerShell.Core/Enter-PSSession.md [hunk 2]
```diff
@@ -868,33 +868,42 @@ The cmdlet does not return any output.
...
+is available. To cancel the `Enter-PSSession` command, press `CTRL+C`.
```
Comment (chasewilson) [inline reference/7.1/Microsoft.PowerShell.Core/Enter-PSSession.md:892]: CTRL + C should be in the format for key presses

File: reference/7.1/Microsoft.PowerShell.Core/Enter-PSSession.md [hunk 3]
```diff
@@ -868,33 +868,42 @@ The cmdlet does not return any output.
...
+
+The **HostName** parameter set was included starting with PowerShell 6.0. It was added to provide
+PowerShell remoting based on Secure Shell (SSH). Both SSH and PowerShell are supported on multiple
+platforms (Windows, Linux, macOS) and PowerShell remoting will work over these platforms where
+PowerShell and SSH are installed and configured. This is separate from the previous Windows only
+remoting that is based on WinRM and much of the WinRM specific features and limitations do not
+apply. For example WinRM based quotas, session options, custom endpoint configuration, and
```
Comment (chasewilson) [inline reference/7.1/Microsoft.PowerShell.Core/Enter-PSSession.md:899]: For example should either be followed by a colon or a comma. Colon would be more common but might look odd? Let me know what you think

File: reference/7.1/Microsoft.PowerShell.Core/Enter-PSSession.md (hunk 1 above)
Comment (sdwheeler) [reply reference/7.1/Microsoft.PowerShell.Core/Enter-PSSession.md:891]: Don't know how that happened. Fixed

File: reference/7.1/Microsoft.PowerShell.Core/Enter-PSSession.md (hunk 2 above)
Comment (sdwheeler) [reply reference/7.1/Microsoft.PowerShell.Core/Enter-PSSession.md:892]: Fixed

File: reference/7.1/Microsoft.PowerShell.Core/Enter-PSSession.md (hunk 3 above)
Comment (sdwheeler) [reply reference/7.1/Microsoft.PowerShell.Core/Enter-PSSession.md:899]: Fixed

**Pull request merged**